src/snippets/get-started/sveltekit/Step3.ts

```diff
@@ -45,5 +39,15 @@ export async function handle({
     }
   }
 
-  return resolve(event);
+  return new Response(
+    JSON.stringify({
```

Comment:
what do we want to return as default?

Reply:
See above. You shouldn't need the additional argument for 200/content-type if you use the helper.

Reply:
should be taken care of now